backend/attendance-service/src/main/java/org/egov/web/controllers/AttendanceLogApiController.java

```diff
@@ -5,7 +5,7 @@
 import org.egov.service.AttendanceLogService;
 import org.egov.web.models.AttendanceLogRequest;
 import org.egov.web.models.AttendanceLogResponse;
-import org.egov.web.models.AttendanceLogSearchCriteria;
+import org.egov.models.AttendanceLogSearchCriteria;
```

Comment:
This can stay in web/models.

Reply:
AttendanceLogSearchCriteria file moved to web/models.